project_stage_closed/models/task_stage.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class ProjectTaskType(models.Model):
+    _inherit = 'project.task.type'
+
+    closed = fields.Boolean(
+        help="Tasks in this stage are considered closed.",
+        default=False)
```

Comment:
The field is not exaclty declared as Odoo (string='close' and help='Tasks in this stage are considered as closed.'). But It's not blocking for me